release-handbook/release_handbook.md

```diff
@@ -46,138 +47,175 @@ This information guides you to archive the documentation of the previous version
 
 The vNext release documentation is hosted in the `docs-staging` branch. The `docs-staging` branch is merged to the `master` branch when a release is published. 
 
-Once a new release rolls out, the `docs-staging` branch should be updated to get prepared for the next release. The updates include but are not limited to the following aspects:
+Once the Code Freeze for a new release takes place, the `docs-staging` branch must be updated to get prepared for the next release. The updates include but are not limited to the following aspects:
 
-- Archive the previous cversion documentation
+- Archive the previous version documentation
 - Bump the release version number to vNext
 - Create placeholder files and folders for the vNext release, such as release notes document and TPSR document
 
-The following steps takes v1.25 release prepration as an example. v1.24 was released a day ago, now we will prepare the `docs-staging` branch for the v1.25 release. An example commit for the steps can be found in [this](https://github.com/zowe/docs-site/commit/7f3dc7a938609d84f15d3ad6e67198155d0de029) commit. 
+The following steps takes v1.25 release preparation as an example. You should follow these steps the day Code Freeze takes place.
+### **Part 1: Sync the doc branches**
 
-### Part 1: Archive the previous release doc
+Summary: Update the `docs-staging` branch so it includes all the necessary updates made to the `master` branch.
 
-1. Update the `docs-staging` branch so it includes all the necessary updates made to the `master` branch. You may need to cherry pick changes to ensure the correct content is copied over to `docs-staging`.
-    - You can use GitHub Desktop to do this. Select the `Branch` menu, then the `Compare to branch` option. Select the `master` branch for comparison.
-2. Go to the `docs-staging` branch.
-3. Navigate to the `/docs-site` directory.
-4. Run the following command:
+**Procedure**
+1. Create a pull request to update the `docs-staging` branch with changes made to the `master` branch.
+  
+    This syncs both branches so that both contain the same content.
+
+2. Create a local build to confirm everything works in the updated `docs-staging` branch.
+
+    1. Run `npm install`.
+    2. Run `npm start` to build the site locally and clear any errors.
+
+Next, let's archive the content for the current version (example: v1.24).
+
+### **Part 2: Archive the previous release doc**
+
+Summary: Create new directories to archive content. Relocate content files for the *current* release (v1.24.x) so they become content files for the *previous* release.
+
+**Procedure**
+
+1. Create and publish a new (temporary) branch based off the `master` branch.
```

Comment:
Are we assuming that the branch from Part 1 has already been merged at this point?

I think either we should do that, or a simpler option may be to reuse the same branch/PR here in Part 2. Trying to avoid any confusion that may arise from having 2 PRs open at the same time for Part 1 and Part 2.

Reply:
it'd be good to make sure there aren't two release-procedure PRs open at the same time.
So, I'm inclined to agree with Timothy on this point as well.
We should probably clarify that the PR in Part 1 should be merged before starting Part 2

Reply:
Edited accordingly.